bundles/org.openhab.binding.bluetooth.airthings/src/main/resources/OH-INF/thing/airthings.xml

```diff
@@ -13,6 +13,7 @@
 
 		<label>Airthings Wave Mini</label>
 		<description>Indoor air quality monitor</description>
+		<semantic-equipment-tag>WeatherStation</semantic-equipment-tag>
```

Comment:
I doubt an indoor sensor can be tagged as weatherstation. it does not have much to do with weather. It is more about climate.  Maybe `AirQualitySensor`

Reply:
Ok. .. I may also review some of the changes from yesterday..